### DISCUSSION TITLE
Is it possible to enable the "rewind" option?

### DISCUSSION THREAD

**Discussion by xxDaVlDxx** [forkymcforkface/RGBPi-Extra#1] [Q&A] [answered]
Is it possible to enable the "rewind" option?

I have been playing with global_overrides.ini but no success.

**Comment by forkymcforkface** [comment]
it is, but will take a little work to get it working properly and stay turned on after exiting a game.

**Comment by forkymcforkface** [comment, marked as answer]
with the RA overrides, rewind can be added.
